### ISSUE TITLE
Blood brother conversions are not explicit enough to the converter

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#80094]
Reporting client version: 514.1589

## Round ID:
[219883](https://scrubby.melonmesa.com/round/219883)

## Reproduction:

Specifically, and perhaps not only limited to, but the code in on_mob_successful_flashed_carbon (https://github.com/tgstation/tgstation/blob/master/code/modules/antagonists/brother/brother.dm#L52) all feedback is exclusively generated through runetext - a system that only displays messages very briefly, might be easy to miss, and perhaps the option to turn off rune text would disable this entirely.

After a round where a player got confused why their conversion never worked (it did, they just didn't pick up on this fact), it became evident this might simply not be enough notification and likely any key events should ALSO generate a reasonable description of progress in the chat log and not assume that runetext alone is sufficient communication.

(I would have assumed they could see the heart HUD marker too but they might have overlooked it or not understood it in some way, or maybe thats not visible too)

(Also there doesn't seem to be any "has gained antag datum xxx" on the converted players PP/Logs listing)
